jina/peapods/pods/k8slib/kubernetes_deployment.py

```diff
@@ -101,6 +103,19 @@ def deploy_service(
         custom_resource_dir=custom_resource_dir,
     )
 
+    kubernetes_tools.create(
+        'configmap',
+        {
+            'name': name,
+            'target': name,
+            'namespace': namespace,
+            'data': envs,
```

Comment:
this means that the defaults in the template are always overriden no? So if the user does not provide any env variables things like JINA_LOG_LEVEL will be missing?

Reply:
PR updated